docs/Operators.md

```diff
@@ -15291,6 +15552,57 @@ expect(node, inputs=[input], outputs=[y for y in expected_outputs], name='test_s
 </details>
 
 
+### <a name="SplitToSequence"></a><a name="splittosequence">**SplitToSequence**</a>
+
+  Split a tensor into a sequence of tensors, along the specified
```

Comment:
Similar to my comment on concat, I think it'd be verbose to realize `list(a_tensor)` with this op. IIUC, we need to loop over the output sequence and apply Squeeze.

Reply:
Added attribute `keepdims` default to `1`. Also making input `split` optional, so that if not provided the split size is 1. `keepdims` will be ignored if `split` is provided, since the only case we need `keepdims` is when split size is 1.